src/settings-renderer/components/menu-preview/MenuPreview.tsx

```diff
@@ -388,8 +388,8 @@ export default function MenuPreview() {
   // rendered as a div with a contained icon and potentially a list of grandchild items.
   const renderChild = (child: RenderedMenuItem, index: number) => {
     let angleDragMayStart = false;
+    let clickedDownContainer: any = null;
```

Comment:
I think ESLint would like this more:


```suggestion
    let clickedDownContainer: HTMLDivElement | null = null;
```